data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,16 @@
+/* exported countValues */
+
+function countValues(stack) {
+  if (stack.peek() === undefined) {
+    return 0;
+  } else {
+    const newArray = [];
+    while (stack.peek() !== undefined) {
+      const newnum = stack.pop();
+      newArray.push(newnum);
+
+    } return newArray.length;
+
```

Comment:
This works, but you could have also used a counter variable to count how many times the loop runs.